File: app/src/main/java/com/softaai/designpatterns/java/creational/builder/meal/item/Item.java
package com.softaai.designpatterns.java.creational.builder.meal.item;

import com.softaai.designpatterns.java.creational.builder.meal.packing.Packing;

/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
public interface Item {

    public String name();
    public Packing packing();
    public float price();

}

File: app/src/main/java/com/softaai/designpatterns/kotlin/creational/builder/meal/item/Item.kt
package com.softaai.designpatterns.kotlin.creational.builder.meal.item

import com.softaai.designpatterns.kotlin.creational.builder.meal.packing.Packing


/**
 * Created by Amol Pawar on 01-04-2019.
 * softAai Apps
 */
interface Item {
    fun name(): String
    fun packing(): Packing
    fun price(): Float
}